agent/app/agent.go

```diff
@@ -196,6 +199,15 @@ func (agent *ecsAgent) printECSAttributes() int {
 	return exitcodes.ExitSuccess
 }
 
+func (agent *ecsAgent) runHealthcheck() int {
+	_, err := http.Get("http://localhost:51678/v1/metadata")
+	if err != nil {
+		seelog.Warnf("Unable to obtain healthcheck: %v", err)
```

Comment:
How about something like "Health check failed with error: ..."

Reply:
yeah, will change it